### PR TITLE
PFM-TASK-3679: Move usage documentation into own subfolder for KB production

### PR DESCRIPTION
**Pull request by falkt**
Resolves [PFM-TASK-3679](https://base.cplace.io/pages/xzj6q0gsyzlpcgf8a0iln6cku/PFM-TASK-3679-Change-production-of-cplace-cli-documentation)

### CODE REVIEW

**Comment by ralfcollabfac** [comment]
@falkt The content that is moved in this pull request is in dire need of an editorial review. Maybe you could create a follow-up pull request for that review. I would volunteer as a reviewer.

File: README.md [hunk 1]
```diff
@@ -17,234 +17,7 @@ This package provides some CLI tools for working with cplace code.
...
-            --specFileRetries The number of times to retry the entire specfile when it fails as a whole
-              (default: 0)
-            --chromeDriverVersion will adjust the version of the Chrome driver
-              (default: latest version)
-
-    Global options:
-        --verbose
-            Print verbose information to console
-```
+The user documentation for the cplace CLI is located [in the Knowledge Base.](/cplace-cli/)
```
Comment (ralfcollabfac) [inline README.md:20]: Since the target is not versioned, you might use a Hugo-specific link here.

Comment (ralfcollabfac) [reply README.md:20]: I see. Since this file is outside of the Knowledge Base, you cannot use a Huo link here.

Comment (falkt) [reply README.md]: But the relative link was also wrong. I added an absolute link to the KB now (it's not working as the target doesn't exist yet).

**Comment by falkt** [comment]
> @falkt The content that is moved in this pull request is in dire need of an editorial review. Maybe you could create a follow-up pull request for that review. I would volunteer as a reviewer.

The content of the page (output of the --help command) is part of the source of the cplace-cli. I think, we should review the text content there first: https://github.com/collaborationFactory/cplace-npm-tools/blob/master/src/cli.ts

File: README.md [hunk 2]
```diff
@@ -17,234 +17,7 @@ This package provides some CLI tools for working with cplace code.
...
-            --specFileRetries The number of times to retry the entire specfile when it fails as a whole
-              (default: 0)
-            --chromeDriverVersion will adjust the version of the Chrome driver
-              (default: latest version)
-
-    Global options:
-        --verbose
-            Print verbose information to console
-```
+The user documentation for the cplace CLI is located [in the Knowledge Base.](https://docs.cplace.io/cplace-cli/)
```
Comment (ralfcollabfac) [inline README.md:20]: I would not use an article with the tool name cplace CLI.
```suggestion
The user documentation for cplace CLI is located [in the Knowledge Base.](https://docs.cplace.io/cplace-cli/)
```

**Review by ralfcollabfac — COMMENTED** [review]
Just one thing.

**Pull request merged**